src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -701,6 +1063,94 @@ internal static UnixTm DateTimeToUnixTm(DateTime date)
                 [DllImport(psLib, CharSet = CharSet.Ansi, SetLastError = true)]
                 internal static extern int GetInodeData([MarshalAs(UnmanagedType.LPStr)]string path,
                                                         out UInt64 device, out UInt64 inode);
+
+                /// <summary>
+                /// This is a struct from getcommonstat.h in the native library. It's a synthetic construct of the Unix stat
```

Comment:
Could you reword `synthetic construct`? It initially confused me until I realised we're saying `this struct doesn't occur anywhere like this, but this type represents the maximal union of all structs in the wild`.

Reply:
done!